nemo/collections/nlp/models/language_modeling/megatron_base_model.py

```diff
@@ -892,7 +892,14 @@ def configure_optimizers(self):
                     ]
                 for bucket in buckets:
                     self._optimizer.init_params_bucket(bucket)
-                self._optimizer.init_params_bucket(self.parameters())
+                try:
+                    params = self.parameters(requires_grad_only=True)
+                except TypeError as e:
```

Comment:
Could you please put more explanations on this change? What does it do and why is it necessary? I suppose it's related to mixed modality training, but we need more explanations to help other people understand the code

Reply:
@pzelasko could you please address this?